hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -197,6 +200,27 @@ public HoodieMetadataTableValidator(JavaSparkContext jsc, Config cfg) {
     this.taskLabels = generateValidationTaskLabels();
   }
 
+  /**
+   * Returns list of Throwable which were encountered during validation. This method is useful
+   * when ignoreFailed parameter is set to true.
+   */
+  public List<Throwable> getThrowables() {
+    return throwables;
+  }
+
+  /**
+   * Returns true if validation has failed. This method is useful when ignoreFailed
+   * parameter is set to true.
+   */
+  public boolean isValidationFailed() {
```

Comment:
Nitpick: method name could be `hasValidationFailure()` to distinguish between generic failures and validation failures?

Reply:
Addressed